Resources/Prototypes/Entities/Structures/base_structure.yml

```diff
@@ -45,12 +46,13 @@
     fixtures:
     - shape:
         !type:PhysShapeAabb
-          bounds: "-0.45,-0.45,0.45,0.45"
+        bounds: "-0.45,-0.45,0.45,0.45"
       mass: 50
-      layer:
-      - SmallImpassable
       mask:
-      - VaultImpassable
+      - MachineMask
+      layer:
+      - MidImpassable
+      - LowImpassable
```

Comment:
why mid/low impassable for base structure?

Reply:
Because most of the inheritors that don't overwrite Fixture don't have damageable and can for the most part be flown over. Though, I couldn't really decide on what flying mobs should collide with so need to discuss that.